mutiny-core/src/nodemanager.rs

```diff
@@ -306,10 +307,50 @@ impl From<bdk::TransactionDetails> for TransactionDetails {
     }
 }
 
+/// Information about a channel that was closed.
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq, Hash)]
+pub struct ChannelClosure {
+    pub user_channel_id: Option<[u8; 16]>,
+    pub channel_id: Option<[u8; 32]>,
+    pub node_id: Option<PublicKey>,
+    pub reason: String,
+    pub timestamp: u64,
```

Comment:
this uses `timestamp` while others uses `last_updated`, not sure if you want to be consistent here.

Reply:
Same here, backwards compatibility